src/gsy_e/models/strategy/load_hours.py

```diff
@@ -190,6 +189,10 @@ def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None,
         self._cycled_market = set()
         self._simulation_start_timestamp = None
 
+    @classmethod
+    def _create_energy_parameters(cls, *args):
```

Comment:
Is there a reason for encapsulating it into a class method? AFAIS we can initialize the `LoadHoursEnergyParameters` on the spot in the constructor

Reply:
Same as the previous comment, this method is deleted in a followup commit.